test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+odoo14-addon-partner-tz @ git+https://github.com/trobz/partner-contact@14.0-mig-partner_tz#subdirectory=setup/partner_tz
```

Comment:
@trobz can you add into your PR description than this one depends on others PR?

Reply:
@lmignon ok, I updated the PR description

Reply:
it depends on this as well https://github.com/OCA/server-tools/pull/1988 (I've added it to description)